be/src/exec/orc_scanner.cpp

```diff
@@ -428,4 +442,38 @@ void ORCScanner::close() {
     _row_reader.reset(nullptr);
 }
 
+void ORCScanner::build_name_id_map() {
+    std::vector<std::string> columns;
```

Comment:
We should clear _map_column_to_id first by invoke `_map_column_to_id.clear()` here.